terraform/workspace-variables/staging_app_env.yml

```diff
@@ -6,6 +6,7 @@ DFE_SIGN_IN_REDIRECT_URL: https://staging.teaching-vacancies.service.gov.uk/auth
 DFE_SIGN_IN_URL: https://pp-api.signin.education.gov.uk
 DISABLE_EXPENSIVE_JOBS: true
 DOMAIN: staging.teaching-vacancies.service.gov.uk
+ENFORCE_LOCAL_AUTHORITY_ALLOWLIST: false
```

Comment:
I think it should be `true` here in the interests of keeping staging as close to production as reasonably possible